examples/webgpu_postprocessing_pixel.html

```diff
@@ -0,0 +1,290 @@
+
+		import { pass, mrt, output, normalView, uniform, directionToColor } from 'three/tsl';
+
+		let camera, scene, renderer, postProcessing, crystalMesh, clock;
+		let gui, effectController;
+
+		init();
+
+		function init() {
+
+			const aspectRatio = window.innerWidth / window.innerHeight;
+
+			camera = new THREE.OrthographicCamera( - aspectRatio, aspectRatio, 1, - 1, 0.1, 10 );
+			camera.position.y = 2 * Math.tan( Math.PI / 6 );
+			camera.position.z = 2;
+
+			scene = new THREE.Scene();
+			scene.background = new THREE.Color( 0x151729 );
+
+			clock = new THREE.Clock();
+
+			// textures
+
+			const loader = new THREE.TextureLoader();
+			const texChecker = pixelTexture( loader.load( 'textures/checker.png' ) );
+			const texChecker2 = pixelTexture( loader.load( 'textures/checker.png' ) );
+			texChecker.repeat.set( 3, 3 );
+			texChecker2.repeat.set( 1.5, 1.5 );
+
+			// meshes
+
+			const boxMaterial = new THREE.MeshPhongMaterial( { map: texChecker2 } );
+
+			function addBox( boxSideLength, x, z, rotation ) {
+
+				const mesh = new THREE.Mesh( new THREE.BoxGeometry( boxSideLength, boxSideLength, boxSideLength ), boxMaterial );
+				mesh.castShadow = false;
+				mesh.receiveShadow = true;
+				mesh.rotation.y = rotation;
+				mesh.position.y = boxSideLength / 2;
+				mesh.position.set( x, boxSideLength / 2 + .0001, z );
+				scene.add( mesh );
+				return mesh;
+
+			}
+
+			addBox( .4, 0, 0, Math.PI / 4 );
+			addBox( .5, - .5, - .5, Math.PI / 4 );
+
+			const planeSideLength = 2;
+			const planeMesh = new THREE.Mesh(
+				new THREE.PlaneGeometry( planeSideLength, planeSideLength ),
+				new THREE.MeshPhongMaterial( { map: texChecker } )
+			);
+			planeMesh.receiveShadow = true;
+			planeMesh.rotation.x = - Math.PI / 2;
+			scene.add( planeMesh );
+
+			const radius = .2;
+			const geometry = new THREE.IcosahedronGeometry( radius );
+			crystalMesh = new THREE.Mesh(
+				geometry,
+				new THREE.MeshPhongMaterial( {
+					color: 0x68b7e9,
+					emissive: 0x4f7e8b,
+					shininess: 10,
+					specular: 0xffffff
+				} )
+			);
+			crystalMesh.receiveShadow = false;
+			crystalMesh.castShadow = false;
+			scene.add( crystalMesh );
+
+			// lights
+
+			const directionalLight = new THREE.DirectionalLight( 0xfffecd, 1.5 );
+			directionalLight.position.set( 100, 100, 100 );
+			directionalLight.castShadow = true;
+			directionalLight.shadow.mapSize.set( 2048, 2048 );
+			scene.add( directionalLight );
+
+			const fillLight = new THREE.DirectionalLight( 0x2a3c6b, 1.2 );
+			fillLight.position.set(-100, 100, 100);
+			fillLight.castShadow = true;
+			fillLight.shadow.mapSize.set( 2048, 2048 );
+			scene.add( fillLight );
+
+			const spotLight = new THREE.SpotLight( 0xffc100, 10, 10, Math.PI / 16, .02, 2 );
+			spotLight.position.set( 2, 2, 0 );
+			const target = spotLight.target;
+			scene.add( target );
+			target.position.set( 0, 0, 0 );
+			spotLight.castShadow = true;
+			scene.add( spotLight );
```

Comment:
Do you mind creating the example with the exact same lighting conditions than the original version?

It's important to perform a 1:1 comparison so we can see possible deviations.

Reply:
I can apply the same lighting parameters to the scene, but the original lighting can't be replicated due to the shadow artifacts present, as mentioned in #28642. In my testing, recreating the lighting conditions of almost any webgl example that uses standard Three.js lights and casts shadows onto other objects exhibits similar rendering issues.

Reply:
Then ignore the shadow casting for now. However, the type of lights and their parametrization should match otherwise the scene's color tone is different which makes it impossible to review the PR.